make-coe/make-coe.go

```diff
@@ -20,6 +20,27 @@ type MakeCOE struct {
 	Account          *Account
 }
 
+func handleNotAcceptable(err libcarina.HTTPErr) error {
+	return errors.Wrap(err, "Unable to communicate with the Carina API because the client is out-of-date. Update the carina client to the latest version. See https://getcarina.com/docs/tutorials/carina-cli#update for instructions.")
+}
+
+func handleHTTPError(err libcarina.HTTPErr) error {
+	switch err.StatusCode {
+	default:
+		return err
+	case 406:
+		return handleNotAcceptable(err)
+	}
+}
+
+func handleLibcarinaError(err error) error {
```

Comment:
Nope not different, just feeling guilty for not writing tests then expecting you to do a "big bang test suite"  to in order to pass a review... 😄 So I'm leaving it up to you how much time you'd like to keep putting into the story.